docker-compose.dev.yml

```diff
@@ -30,6 +30,11 @@ services:
       HYPERDX_API_KEY: ${HYPERDX_API_KEY}
       HYPERDX_LOG_LEVEL: ${HYPERDX_LOG_LEVEL}
       OPAMP_SERVER_URL: 'http://host.docker.internal:${HYPERDX_OPAMP_PORT}'
+      # Uncomment to enable stdout logging for the OTel collector
+      # OTEL_SUPERVISOR_PASSTHROUGH_LOGS: 'true'
+      # Uncomment to enable JSON schema in ClickHouse
```

Comment:
nit: this should say Comment out to disable JSON schema in clickhouse + update the verbiage below.  Unless this was intended to be enabled by default?